File: huannv_Rx_Retrofit_MultiThread/app/src/main/java/com/example/base_project/base/baseApi/ApiUtils.java
package com.example.base_project.base.baseApi;

public class ApiUtils {
    public static final String BASE_URL = "https://api.themoviedb.org/3/";

    public static ApiRepository getDataApi(){
        return ApiBuilder.getApi(BASE_URL).create(ApiRepository.class);
    }
}

File: Trainning_Kotlin/app/src/main/java/com/example/myapplication/base/baseApi/ApiUtils.kt
package com.example.myapplication.base.baseApi

class ApiUtils {


    companion object {
        var BASE_URL : String = "https://api.themoviedb.org/3/";
        fun getDataApi(): ApiRepository? {
            return ApiBuilder.getApi(BASE_URL).create(ApiRepository::class.javaObjectType)
        }
    }

}